src/lib/getSchemaUiJson.ts

```
@@ -0,0 +1,135 @@
+  if (hasAllLiteralTypes(property)) {
+    // Return the first literal type, prioritizing a non-null type.
+    return uiPropertyLiteralToJson({
+      ...property,
+      types: [property.types.find(type => type !== 'null') ?? 'null'],
+    }, useExample)
+  }
+
+  if (isSingleType(property, 'object')) {
+    if (isOneOfProperty(property)) {
+      // Return the first property of the oneOf array.
+      return uiPropertyToJson(property.properties![0], useExample)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)